### PR TITLE
Enable Broken VI check using VI Analyzer

### PR DESCRIPTION
**Pull request by aepete**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/measurement-services-labview/blob/main/CONTRIBUTING.md). _(Required)_
- [ ] (n/a)  Automatically post PR comments with images for G code changes? _(Recommended for small changes)_

### What does this Pull Request accomplish?

Leverages the VI Analyzer workflow to verify no broken VIs under Source/Runtime

### Why should this Pull Request be merged?

Using VI Analyzer is the [preferred way to do a no-broken-VIs check](https://github.com/ni/measurementlink-labview/pull/458#discussion_r1425967210)...  Previously we relied upon the test VI [test_VerifyVIsNotBroken.vi](https://github.com/ni/measurementlink-labview/pull/460/files#diff-68ac16d033657b9109993e1edbe2603e3093ea9288f7d846503ae4aad49a68c7), which also checked under Source\Runtime for broken VIs.  I've hereby deleted that test to avoid redundancy.

### What testing has been done?

Ran & passed locally.  Ran & passed in PR workflow.

### CODE REVIEW

**Review by pbirkhol-ni — APPROVED** [review]

File: Source/Tests/Tests.All/test_VerifyVIsNotBroken.vi
Comment (pbirkhol-ni) [inline Source/Tests/Tests.All/test_VerifyVIsNotBroken.vi:1]: Is this part of a library? If so, make sure you remove it from the library. However, I suspect it isn't part of a library.

Comment (aepete) [reply Source/Tests/Tests.All/test_VerifyVIsNotBroken.vi:1]: It is not.  I don't think we have many or any libraries for the test VIs.

Comment (aepete) [reply Source/Tests/Tests.All/test_VerifyVIsNotBroken.vi:1]: For posterity, here's what the diagram looked like:

image

Comment (pbirkhol-ni) [reply Source/Tests/Tests.All/test_VerifyVIsNotBroken.vi:1]: > I don't think we have many or any libraries for the test VIs.

That used to be true, but not any longer. Nearly every test VI is in a library. It's how we manage friendship relationships for testing.

Comment (pbirkhol-ni) [reply Source/Tests/Tests.All/test_VerifyVIsNotBroken.vi:1]: Anyway, I doubt this one is since there is no need for any friendship relationships.

Comment (aepete) [reply Source/Tests/Tests.All/test_VerifyVIsNotBroken.vi:1]: > > I don't think we have many or any libraries for the test VIs.
> 
> That used to be true, but not any longer. Nearly every test VI is in a library. It's how we manage friendship relationships for testing.

Good to know :)

**Pull request merged**